src/coordination/replication_instance.cpp

```diff
@@ -39,6 +40,11 @@ auto ReplicationInstance::OnFailPing() -> bool {
   return is_alive_;
 }
 
+auto ReplicationInstance::IsReadyForUUIDPing() -> bool {
+  return std::chrono::duration_cast<std::chrono::seconds>(std::chrono::system_clock::now() - last_check_of_uuid_)
```

Comment:
Wrong check I think. You are just calculating the difference between now and when it is last checked

Reply:
yes correct, I need to add flag